vec/vec-v2x/src/main/resources/vec2/vec_2.0.2.xjb

```diff
@@ -1,1483 +1,2630 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<jxb:bindings xmlns:jxb="https://jakarta.ee/xml/ns/jaxb"
```

Comment:
I think you have regenerated the file? Even when ignoring the whitespaces, I see a lot of diffs.

![image](https://github.com/4Soft-de/harness-model/assets/7572614/18ad95e1-16f8-46c0-9249-52f8fd61dc1d)

And does only this file needs adjustments? What about the .xjb files from VEC 1.1.X and VEC 1.2.X?

Reply:
Yes, that file is generated (from a different source than the previous version). 

- Now: Generated from meta information embedded in VEC XSD.
- Previously: Generated from VEC XMI UML File. 

That is the whole point of this PR. As the old file had a random order (order inherited from the model file) and the new file inherits the order from the XSD (which is alphabetical) the diff is not really useable. However, creating a diffable version is also not possible, because you can't replicate the order from model with XSD alone.

The other files for VEC 1.Y.X can not be changed, because the embedded meta information in the xsd was just introduced with VEC 2.0.2.

Reply:
Alright, got it. Understood something else regarding meta information.